README.org

```diff
@@ -4,154 +4,140 @@
 
 Code examples for interactive explanations of [[https://github.com/magit/transient][transient]].
 
-This guide assumes you have minimal knowledge of Emacs, some programming
-experience in elisp and non-lisp languages, and have at least seen [[https://magit.vc/screenshots/][screenshots]]
-of =magit=.
+This guide assumes you have minimal knowledge of Emacs, some programming experience in elisp and non-lisp languages, and have at least seen [[https://magit.vc/screenshots/][screenshots]] of =magit=.
 
 * How to use
 
-  There are two ways:
+There are two ways:
 
-  - Open this file in Emacs and run examples as literate org.
-  - Install the package to run commands and read their source.  Start with the
-    =tsc-showcase= command.
+- Open this file in Emacs and run examples as literate org.
+- Install the package to run commands and read their source.  Start with the
+=tsc-showcase= command.
 
 ** Using as literate org document
 
-  If you open this file in Emacs, it will switch to Org mode and you can run
-  individual source blocks with =org-babel-execute-src-blk= on the block.
+If you open this file in Emacs, it will switch to Org mode and you can run individual source blocks with =org-babel-execute-src-blk= on the block.
 
 ** Using as an installable package
 
-  If you install the package, you can read source for each example with the
-  normal.  =describe-symbol= command.  All commands are under =tsc-*= prefix.
-  Somewhat useful suffixes are under =tsc-suffix-*= while less useful ones under
-  =tsc--suffix-*=.  They will come in handy when you are developing new
-  applications.
+If you install the package, you can read source for each example with the normal.  =describe-symbol= command.  All commands are under =tsc-*= prefix.  Somewhat useful suffixes are under =tsc-suffix-*= while less useful ones under =tsc--suffix-*=.  They will come in handy when you are developing new applications.
 
-  Installations for straight and elpaca:
+Installations for straight and elpaca:
 
-  #+begin_src elisp
+#+begin_src elisp
 
-    ;; using straight use-package with custom recipe
-    (use-package transient-showcase
-      :straight '(transient-showcase
-                  :type git :host github :repo "positron-solutions/transient-showcase"))
+;; using straight use-package with custom recipe
+(use-package transient-showcase
+:straight '(transient-showcase
+:type git :host github :repo "positron-solutions/transient-showcase"))
 
-    ;; using elpaca (recommended to add a hash for reproducibility)
```

Comment:
Well, if only it were so easy :D   Will be looking for a second attempt.  Please keep indentation in code blocks wherever it makes sense.

Reply:
How about we just re-indent all the code blocks?

Reply:
Oops forgot to protect code blocks.  It actually wasn't that easy since lists all needed to be rejoined.  I re-indented all code blocks; take another look.